pkg/controller/controller.go

```diff
@@ -58,6 +58,7 @@ type Service struct {
 	Fs fs.Interface
 
 	externalAccess string
+	nfsAcls        string
```

Comment:
should we name this nfs4Acls?

Reply:
wanted to keep it generic as we are using the same field to pass the unix mode bits as well for nfsv3